src/electronic-ids/pcsc/pcsc-common.hpp

```diff
@@ -24,7 +24,9 @@
 
 #include "electronic-id/electronic-id.hpp"
 
-#include "pcsc-cpp/pcsc-cpp-utils.hpp"
+#include <pcsc-cpp/pcsc-cpp-utils.hpp>
```

Comment:
Until now we have used `<` to include system library headers and `"` for including our library headers. I would not change this only in a single header, so I'll revert this. But feel free to discuss this further!